tests/models/gpt2/test_modeling_gpt2.py

```diff
@@ -832,7 +832,8 @@ def test_gpt2_sample_max_time(self):
         start = datetime.datetime.now()
         model.generate(input_ids, do_sample=False, max_time=None, max_length=256)
         duration = datetime.datetime.now() - start
-        self.assertGreater(duration, datetime.timedelta(seconds=1.5 * MAX_TIME))
+        self.assertGreater(duration, datetime.timedelta(seconds=MAX_TIME))
+        self.assertLess(duration, datetime.timedelta(seconds=1.5 * MAX_TIME))
```

Comment:
Only failing test without this modification. I'm not sure if this is how it is intended. Maybe `assertGreater` should be `assertLess` instead?

Reply:
I think that this change is totally fine, now SDPA is used so the generation is faster